### PR TITLE
Removing unused tests; marking health check as "is secure" device check

### PR DESCRIPTION
**Pull request by privateducky**
1. Ensure all tests are falling into our antivirus/malware category OR health check OR emerging threats.
2. Converting our core health check test into a "is it a secure device" check

### CODE REVIEW

**Comment by bfuzzy1** [comment]
We should probably rewrite the health check README to account for the new changes.

**Review by bfuzzy1 — CHANGES_REQUESTED** [review]
Need to rewrite health check README to account for this update.

**Comment by privateducky** [comment]
> Need to rewrite health check README to account for this update.

Great point. I'll do that; and pull this test into a separate PR all together.

**Review by gerbsec — APPROVED** [review]
Tested fully across all tests, all working as intended besides a capitalization fix that I pushed up.

**Review by bfuzzy1 — APPROVED** [review]

File: endpoint/endpoint.go [hunk 1]
```diff
@@ -179,3 +117,24 @@ func IsAvailable(programs ...string) bool {
...
+func isSecure() bool {
+	if _, err := os.Stat("/.dockerenv"); err == nil {
+        return true
+    } else if runtime.GOOS == "ios" {
+		return true
+	} else if runtime.GOOS == "android" {
+		return true
+	}
+	return false
+}
```
[the remark this answers is not in the document]
Comment (privateducky) [reply endpoint/endpoint.go]: agreed; what GOOS is that?

**Pull request merged**